recipes/process-assert-to-assert/src/workflow.ts

```diff
@@ -0,0 +1,121 @@
+import type { Edit, Range, Rule, SgNode, SgRoot } from "@codemod.com/jssg-types/main";
+import { getNodeRequireCalls } from "@nodejs/codemod-utils/ast-grep/require-call";
+import { getNodeImportStatements } from "@nodejs/codemod-utils/ast-grep/import-statement";
+import { resolveBindingPath } from "@nodejs/codemod-utils/ast-grep/resolve-binding-path";
+import { removeBinding } from "@nodejs/codemod-utils/ast-grep/remove-binding";
+import { removeLines } from "@nodejs/codemod-utils/ast-grep/remove-lines";
+
+/**
+ * Transforms deprecated `process.assert` usage to the standard `assert` module.
+ * 
+ * Transformations:
+ * 1. Replaces all `process.assert` references with `assert`
+ * 2. Adds the necessary import/require statement if not already present:
+ *    - For ESM or files without require calls: adds `import assert from "node:assert"`
+ *    - For CommonJS (.cjs files or files using require): adds `const assert = require("node:assert")`
+ * 
+ * Examples:
+ * 
+ * Before:
+ * ```js
+ * process.assert(value);
+ * process.assert.strictEqual(a, b);
+ * ```
+ * 
+ * After:
+ * ```js
+ * import assert from "node:assert";
+ * assert(value);
+ * assert.strictEqual(a, b);
+ * ```
```

Comment:
Remove process.assert.strictEqual reference, I think it was added by mistake. This scenario does not exist. Add bold formatting to the before/after text.
```suggestion
 * **Before**:
 * ```js
 * process.assert(value);
 * ```
 * 
 * **After**:
 * ```js
 * import assert from "node:assert";
 * assert(value);
 * ```
```

Reply:
My point is that the TypeScript docs say that the code in the before block, `process.assert.strictEqual(a, b)`, will be updated to use `assert.strictEqual`.

However, `process.assert.strictEqual` does not exist, at least I couldn’t find any reference to it.